cyences_app_for_splunk/appserver/static/cs_overview.js

```diff
@@ -66,7 +66,7 @@ require([
         { token: 'vulnerability', associated_products: ['Qualys', 'Tenable', 'Nessus', 'CrowdStrike Spotlight'] },
         { token: 'ad_windows', associated_products: ['Sysmon', 'Windows', 'Windows AD', 'Windows DNS'] },
         { token: 'credential_compromise', associated_products: ['Sysmon'] },
-        { token: 'ransomware', associated_products: ['Sysmon', 'Windows', 'Cisco IOS', 'FortiGate', 'Palo Alto', 'Sophos Firewall', 'Cisco Meraki'] },
+        { token: 'ransomware', associated_products: ['Ransomware', 'Sysmon'] },
```

Comment:
is "TOR Traffic detected by Firewall" alert is falling under ransomware? previously, It was under ransomware. if so then we have to add following products as well.

'Windows', 'Cisco IOS', 'FortiGate', 'Palo Alto', 'Sophos Firewall'

Reply:
No, its not specific to Ransomware. its generic security issue on Network.